### PR TITLE
Add new start page copy for means assessment journey

### PR DESCRIPTION
**Pull request by willmcb**
## Description of change

Updates start page copy to match means journey designs.

Is behind feature flag awaiting wider means journey release. Once this has happend, PR can be raised to remove the defunct copy when the feature flag is removed.

## Link to relevant ticket

[CRIMAP-722](https://dsdmoj.atlassian.net/browse/CRIMAP-722)

## Notes for reviewer

Please see ticket for figma designs and double check that the copy is the same.

### Before changes:
Screenshot 2023-11-02 at 14 28 17

### After changes:
Screenshot 2023-11-02 at 14 27 46

## How to manually test the feature
- make sure your meanse journey feature flag is on so as to see the new copy on the start page.
- check copy etc.
- set the feature flag to false --> then make sure current non-means copy shows instead.

[CRIMAP-722]: https://dsdmoj.atlassian.net/browse/CRIMAP-722?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by timpeat — APPROVED** [review]
👍

File: app/views/home/index.en.html.erb [hunk 1]
```diff
@@ -8,20 +8,41 @@
       <%= service_name %>
     </h1>
 
-    <p class="govuk-body">
+    <% if FeatureFlags.means_journey.enabled? %>
+      <p class="govuk-body">
+      Use this service to apply for criminal legal aid in ongoing cases. The application will be means tested.
+      </p>
+      <p class="govuk-body">
+      Your client has a partner.
```
Comment (hiboabd) [inline app/views/home/index.en.html.erb:16]: This should say your client cannot have a partner 🙂 : https://www.figma.com/file/thkvnDBbQbHlSqzt1TE3lk/Crime-Apply?type=design&node-id=7670-22054&mode=design&t=SuTbkhVAFzZlXix0-0

**Review by hiboabd — CHANGES_REQUESTED** [review]
Small change, otherwise tested locally and looks good

File: app/views/home/index.en.html.erb (hunk 1 above)
Comment (willmcb) [reply app/views/home/index.en.html.erb:16]: Thanks will amend! :)

**Review by hiboabd — APPROVED** [review]

**Pull request merged**